docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.3'
+services:
+  xu:
+    image: exchangeunion
+    build:
+      context: .
+    volumes:
+      - ./:/opt/xud
+      - ./xud.conf:/home/node/.xud
```

Comment:
See my comment below on whether we actually need to create an xud.conf file.

Reply:
We need to be able to inject the MySQL container connection details. hence, I think this is required.